### PR TITLE
Resnponsive Portolio Page

### PR DESCRIPTION
**Pull request by Besufekad-HAZ**
-add a mobile-first portfolio page
-add  responsiveness 
-add media queries
-add mobile dropdown menu
-add contact form validation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,6 +29,327 @@
...
+            <ul>
+             <li><a class="nv" href="#Hello">Hello</a></li>
+            <li><a class="nv" href="#Portfolio">Portfolio</a></li>
+            <li><a class="nv" href="#About_sec">About</a></li>
+            <li><a class="nv" href="#Contact">Contact</a></li>
+            </ul>
+          </div>
+        </div>
+
+
```
Comment (bellom) [inline index.html]: It would be nice to indent your code probably and remove too many line spaces. Do this in this file.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,902 @@
...
+/* @media screen and (max-width: 768px) { */
+body {
+  overflow-x: hidden;
+}
+
+/* * {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+} */
```
Comment (bellom) [inline style.css]: Kindly remove all commented code in this fiile.

**Review by bellom — CHANGES_REQUESTED** [review]
@Besufekad-HAZ 

### This is a manual review

# Request Changes

Cheers and Happy coding!:clap::clap::clap:

**Comment by Besufekad-HAZ** [comment]
Dear sir,

Thanks for the review. I have made the required changes and pushed them to
the pull request link below.

https://github.com/Besufekad-HAZ/My-Portfolio/pull/2


Best regards,

Besufekad Alemu.

**Pull request merged**